tests/library/precompiled-dxil-generics.slang

```diff
@@ -0,0 +1,28 @@
+// precompiled-dxil-generics.slang
+
+// A test that uses slang-modules with embedded precompiled DXIL and a library containing generics.
+// The test compiles a library slang (export-library-generics.slang) with -embed-dxil then links the
+// library to entrypoint slang (this file).
+// The test passes if there is no errror thrown.
+// TODO: Check if final linkage used only the precompiled dxil.
+
+//TEST(windows):COMPILE: tests/library/export-library-generics.slang -o tests/library/export-library-generics.slang-module -embed-dxil -profile lib_6_6 -incomplete-library
```

Comment:
I don't think `(windows)` here is doing anything? Why do we need it?

Reply:
It is better to use `TEST:SIMPLE(filecheck=CHECK): ...` and use filecheck to verify the compiled results, then just check for the return code of a compile.

Reply:
-embed-dxil only works on windows